### ISSUE TITLE
Crash if Python is not installed

### ISSUE THREAD

**Issue by JernejHabjan** [getnamo/tensorflow-ue4#47]
Hey!

I've come to receive crash at breakpoint seen on image, if there is no python installed on system.
I tried this using releases for 4.21, 4.20.
Maybe some error should be thrown and caught exception for initialize threads or something.
Hope it helps.

![image](https://user-images.githubusercontent.com/21958321/56590413-5a661100-65e7-11e9-8314-d639ca675821.png)

**Comment by getnamo** [comment]
The python installation should be provided with the releases and should have no dependency on what's installed on your system, are you clicking the .7z links in e.g. https://github.com/getnamo/tensorflow-ue4/releases/tag/0.11.0 when you are downloading a release? Source code (zip) links do not included necessary binaries for embedded python.

**Comment by JernejHabjan** [comment]
Yes, I downloaded tensorflow-ue4.21-v0.11.0-cpu.7z zip and extracted it in project directory as intended.
Also used UE4 4.21 on freshly installed computer and VS 2017 with 'C++ game development' install bundle.

Ok I recreated the issue.
- Cleaned project (saved, binaries, intermediate), also removed plugins folder and redownloaded .7z and extracted there.
- Removed Python36 from environmental variables
- Removed Python36 folder from programFiles
- Launched Project, which caused full rebuild.

Crash occurred at same spot as before.
And then I tried adding Python36 folder back to programFiles and launched project and viola - it worked! - no crash this time.

It seems it takes python install from programFiles when launching editor.

hope this helps 😄
